rma/wizards/rma_make_picking.py

```diff
@@ -34,15 +34,22 @@ def default_get(self, fields_list):
         context = self._context.copy()
         res = super(RmaMakePicking, self).default_get(fields_list)
         rma_line_obj = self.env["rma.order.line"]
-        rma_line_ids = self.env.context["active_ids"] or []
+        rma_obj = self.env["rma.order"]
+        active_ids = self.env.context["active_ids"] or []
         active_model = self.env.context["active_model"]
-
-        if not rma_line_ids:
+        if not active_ids:
             return res
-        assert active_model == "rma.order.line", "Bad context propagation"
+        assert active_model in [
+            "rma.order.line",
+            "rma.order",
+        ], "Bad context propagation"
 
         items = []
-        lines = rma_line_obj.browse(rma_line_ids)
+        if active_model == "rma.order":
+            rma = rma_obj.browse(active_ids)
+            lines = rma.rma_line_ids.filtered(lambda x: x.state == "approved")
+        else:
```

Comment:
I don't think all approved lines should always be added when coming from a group.
I'd say, if the highlighted button is pressed, then we should only get lines having a qty_to_receive > 0
(maybe passing a context on the button for instance ?)
And if the not highlighted button is pressed, then all lines with a receipt policy != 0?

Reply:
@florian-dacosta , it's done.